ui/app/manager/src/pages/page-users.ts

```diff
@@ -804,6 +809,22 @@ export class PageUsers extends Page<AppStateKeyed> {
                                                   @or-mwc-input-changed="${(e: OrInputChangedEvent) => user.enabled = e.detail.value}"
                                                   style="height: 56px;"></or-mwc-input>
 
+                                    <!-- realmRoles roles -->
+                                    <or-mwc-input
+                                            ?readonly="${readonly}"
+                                            ?disabled="${isSameUser}"
+                                            .value="${user.realmRoles && user.realmRoles.length > 0 ? user.realmRoles.filter(r => !this._realmRolesToExclude.includes(r.name)).filter(r =>!r.composite).map(r => r.name) : undefined}"
+                                            .type="${InputType.SELECT}" multiple
+                                            .options="${realmRoleOptions}"
+                                            .label="${i18next.t("realm_role_plural")}"
+                                            @or-mwc-input-changed="${(e: OrInputChangedEvent) => {
+                                                const roleNames = e.detail.value as string[];
+                                                user.realmRoles = this._realmRoles.filter(cr => roleNames.some(name => cr.name === name)).map(r => {
```

Comment:
This looks like it will remove the excluded roles from the user which it should not @DonWillems do I need to resolve this ready for tomorrow?

Reply:
Resolved now